advanced/Scripts/piholeDebug.sh

```diff
@@ -230,7 +230,7 @@ initialize_debug() {
 
 # This is a function for visually displaying the current test that is being run.
 # Accepts one variable: the name of what is being diagnosed
-# Colors do not show in the dasboard, but the icons do: [i], [✓], and [✗]
+# Colors do not show in the dashboard, but the icons do: [i], [✓], and [✗]
```

Comment:
Thanks, but actually this comment is outdated and it's not true anymore.

The current web interface shows colors on the output.

Can you please remove lines 233 and 236 (`# These colors do not show in the GUI, but the formatting will`)?

Reply:
Sure thing!